sdk/executor/src/main/java/com/mesosphere/sdk/executor/ProcessTask.java

```diff
@@ -251,4 +258,61 @@ private boolean waitExit(final Duration timeout) {
         }
     }
 
+    /**
+     * Sets up the config files on the executor side.
+     *
+     * @param taskInfo The {@link TaskInfo} to extract the config file data from
+     * @throws IOException if the data in the taskInfo is not valid or the config can't be written to disk
+     */
+    private static void setupConfigFiles(TaskInfo taskInfo) throws IOException {
```

Comment:
Yeah this is really just moving it out of the commons package since it's not used by the scheduler.